lib/portlayer/exec/base.go

```diff
@@ -361,21 +361,24 @@ func (c *containerBase) shutdown(op trace.Operation, waitTime *int32) error {
 	}
 
 	cs := c.ExecConfig.Sessions[c.ExecConfig.ID]
-	stop := []string{cs.StopSignal, string(ssh.SIGKILL)}
-	if stop[0] == "" {
-		stop[0] = string(ssh.SIGTERM)
+	stopActions := []string{"kill", "groupKill"}
```

Comment:
Should these be defined as constants somewhere? I see they're used in a switch statement below.